Modules/SubModule_ASL/xASL_wrp_RealignASL.m

```diff
@@ -216,20 +223,20 @@ function xASL_wrp_RealignASL(x,bSubtraction)
 %% ----------------------------------------------------------------------------------------
 %% 3) Threshold-free spike definition (based on ENABLE, but with t-stats rather than the threshold p<0.05)
 
-bExecutedENABLE = 0;
 
 if bSubtraction && nFrames<=10

```

Comment:
Now at multiple locations in this wrapper, we check for a condition to set bENABLE. Would be good to put all these conditions on the top at a single location, for more overview. E.g., if bSubtraction is false, you would never run bENABLE or bZigZag, and then you don't have to check bMultiTE or bMultiPLD. So that could go to the top. Good to keep this in a logic order, so it's easy to read and debug.